sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,157 @@
+# June'20 release notes
+
+Release notes for the commit range ba404be..24726df
+
+## New features
+  - Added switch to assume that each amount of work-items in each ND-range
+    dimension if less that 2G (fits in signed integer), which allows underlying
+    BEs to perform additional optimizations [fdcaeae] [08f8656]
+  - Added partial support for [Host task with interop capabilities extension](https://github.com/codeplaysoftware/standards-proposals/blob/master/host_task/host_task.md) [ae3fd5c]
+  - Added support for [SYCL_INTEL_bitcast](doc/extensions/Bitcast/SYCL_INTEL_bitcast.asciidoc)
+    as a `sycl::detail::bit_cast` [e3da4ef]
+  - Introduced the Level Zero plugin which enables SYCL working on top of Level0
+    API. Interoperability is not supportet yet [d32da99]
+  - Implemented [parallel_for simplification extension](doc/extensions/ParallelForSimpification) [13fe9fb]
+  - Implemented [SYCL_INTEL_enqueue_barrier extension](doc/extensions/EnqueueBarrier/enqueue_barrier.asciidoc) [da6bfd0]
+  - Implemented [SYCL_INTEL_accessor_simplification extension](https://github.com/intel/llvm/pull/1498) [1f76efc]
+  - Implemented OpenCL interoperability API following [SYCL Generalization proposal](https://github.com/KhronosGroup/SYCL-Shared/blob/master/proposals/sycl_generalization.md) [bae0639]
+
+## Improvements
+### SYCL Frontend and driver changes
```

Comment:
I think we should rename this section to "SYCL compiler" and next section to "SYCL library".

Reply:
Done